src/services/redisSessionStorage.ts

```diff
@@ -83,7 +83,16 @@ export default class RedisSessionStorage extends RedisStorageUtils
       user.fiscal_code
     );
 
-    const removeOtherUserSessionsPromise = this.removeOtherUserSessions(user);
+    // Read ENV to allow multiple user's sessions functionality
+    // Default value is false when the ENV var is not provided
+    const ALLOW_MULTIPLE_SESSIONS = fromNullable(
```

Comment:
I'd rather put this as a flag / parameter to the RedisSessionStorage class so:
- you will refer to the process.env var only at top level
- we will pass the flag explicitely (clarify testing, now we have an "hidden" side effect here in the set() method)

Reply:
Sure, initially I've use this approach to avoid conflict with the awilix-removal branch #520 
I'll resolve the conflict later.
Fixed in ba974fe
An explicit test was added in 7420b60